FALSEHOODS.md

```diff
@@ -135,9 +135,59 @@ surprising how many false assumptions programmers continue to make about them.
 
 1.  **Phone numbers contain only digits**
 
-    In Israel, certain advertising numbers start with a `*`.
+    In Israel, certain advertising numbers start with a `*`. In New Zealand,
+    non-urgent traffic incidents can be reported by calling `*555` from a mobile
+    phone.
 
 1.  **Phone numbers are always written in ASCII**
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  **Phone numbers have only one prefix (area code or national destination code) at a given time**
+
+    In the mid-90s in Iceland, phone numbers changed from 5 and 6 digits to 7 digits.
+    During the transition period, phone numbers could be reached by the old area code
+    or the new 7 digit number (a different prefix).
+
+    **Tip:** During transition periods, make sure that both forms of a number
+    are supported for look-up by phone number. If sending messages/dialling
+    affected numbers, both the new and old prefix may need to be tried.
+    Depending on the type of transition, you may need to contact affected
+    customers and ask them to update their number.
+
+1.  **The leading zero in domestic numbers can always be discarded when dialing
+    internationally**
+
+    In some countries, people write the national prefix in brackets (typically
+    `(0)`) after the country calling code to indicate that it should be
+    discarded when dialing internationally. In Italy, since 1988, the prefix was
+    "fixed" to the phone numbers, so that `(01) 2345` became `012345` and should
+    be dialed (internationally) as `+39012345` (including the leading zero).
+
+    **Tip:** Use the library to parse and format numbers to handle this
+    correctly for every country.
+
+1.  **The country or area code of a phone number indicates the user's location,
+    place of residence, time-zone, or preferred language**
+
+    There are many reasons for someone to have a phone number with an area code
```

Comment:
The area code bit makes it sound like you can at least work out which country they are in. I would say "There are many reasons for someone to have a phone number issued by a state or region other than where they reside or hold citizenship.

These reasons include, but are not limited to:
  * Moving within a country. In countries with phone number portability you may retain your number when moving, even if it is a fixed-line number and even if it has an area code. (example bla)
  * Moving to another country. In this case you probably can't keep a fixed-line, but you could keep your mobile phone.
  * Having many customers from a neighbouring country.
  * Wanting cheaper calling rates. (e.g. voip, people travelling around Europe)
  * etc etc 

Note that geographical area codes are assigned in some countries to mobile phones.